tests/client/util.rs

```diff
@@ -165,7 +180,7 @@ async fn init_drawbridge(oidc_addr: String) -> (u16, Sender<()>, JoinHandle<()>)
             OidcConfig {
                 label: "test-label".into(),
                 issuer: oidc_addr.parse().unwrap(),
-                client_id: "test-client-id".into(),
+                client_id: "4NuaJxkQv8EZBeJKE56R57gKJbxrTLG2".into(),
```

Comment:
this chunk does not match the PR or patch description

Reply:
Updated the description. This makes the mock respect our hardcoded OIDC client key; see src/cli/user/register.rs .